apps/expo-payments/package.json

```diff
@@ -1,5 +1,7 @@
 {
   "main": "__generated__/AppEntry.js",
+  "name": "expo-payments",
```

Comment:
1. Could you move these files to the top of JSON?
2. Adding this will result in [an unlinked dependency warning](https://github.com/expo/expo/blob/master/packages/expo-yarn-workspaces/bin/check-workspace-dependencies.js) (which we in general try to avoid). (`expokit` will be downloaded from NPM) I guess this is just for the `until after-SDK34-release` time period, so maybe we're ok with it? Or not? cc @ide

Reply:
I think it is OK temporarily, but not in the long-term. Aside from the `templates` directory, we want to always be testing master, and therefore all linkable packages should be linked.

Reply:
Sure: 68b9a2aa9705f6d44fb23069377955e6568cb5e9